go/go-introduction/if-statements/if-else.md

```diff
@@ -33,6 +33,13 @@ fmt.Println(x, " is negative")
 // 1 is negative
 ```
 
+The error would be:
+```go
+undefined: x
+```
+
+This is because the `x` we've created inside the `if` statement is only visible to the scope of that `if`. While our second `fmt.Println` that uses the `x` is located outside of the if.
```

Comment:
We should remove those output comments in the code, right? They imply that the code gives a result when it actually errors.

Something like:

````md
```go
if x := 1 ; x > 0 {
// ^ this x is only visible inside the if
  fmt.Println(x, " is positive")
}
fmt.Println(x, " is negative")
//                ^ this will be an error
```
````

Also, shall we simplify?

```suggestion
This is because the `x` created inside the `if` statement is not visible outside of it.
```

Reply:
I forgot to remove the comments. 

I've applied both of your suggestions  for simplification and the comments one